swedish-oidc-profile.md

```diff
@@ -212,8 +209,66 @@ An OpenID Provider compliant with this profile SHOULD support the PKCE extension
 
 An OpenID Provider MUST NOT allow a Relying Party to use the `plain` code challenge method.
 
+<a name="authentication-request-parameter-extensions"></a>
+### 2.2. Authentication Request Parameter Extensions
+
+This section contains authentication request parameter extensions defined within the scope of this profile.
+
+<a name="client-provided-user-message"></a>
+#### 2.3.1. Client Provided User Message
+
+**Parameter:** `https://claims.oidc.se/1.0/userMessage`
```

Comment:
Question: Should custom request parameters really be scoped under the "claims.oidc.se" domain? The term "claim" doesn't really seem to be used in the context of OIDC requests in the core spec, expect for stating what claims the requestee wants in the ID token.

I guess the same applies to the description text, i.e. should the text use "When the user message **parameter** is included ..." rather than "When the user message **claim** is included ..."?

On the other hand - I do understand that having a single `https://claims.oidc.se` used for both custom claims, scopes, request parameters also makes a lot of sense.

Reply:
I had the same thought, but in fact, everything is claims. I am not that keen of having separate domains for all kind of types we define. A suggestion: We use `https://id.oidc.se` as the prefix, where `id` stands for identifier. That may be better!